Source/Documentation/Manual/cabs.rst

```diff
@@ -1364,6 +1364,43 @@ Customizations for such file are possible following these rules:
 Except for the first column, fields 
 in the 3D distributed power display are always with center alignment.
 
+ETCS circular speed gauge and the Driver Machine Interface
```

Comment:
There is already an entry describing the ETCS display. Could you merge it with this?

Reply:
Yeah, it's better that way. Merged.